pyirf/io/gadf.py

```diff
@@ -260,6 +260,64 @@ def create_background_2d_hdu(
     return BinTableHDU(bkg, header=header, name=extname)
 
 
+@u.quantity_input(
+    background=GADF_BACKGROUND_UNIT, reco_energy_bins=u.TeV, fov_offset_bins=u.deg,
+)
+def create_background_3d_hdu(
+    background_3d,
+    reco_energy_bins,
+    fov_lon_bins,
+    fov_lat_bins,
+    alignment = "ALTAZ",
```

Comment:
Maybe define an enum for the alignment or at least check that it is one of the supported values.